lib/trento_web/openapi/schema/database.ex

```diff
@@ -0,0 +1,80 @@
+defmodule TrentoWeb.OpenApi.Schema.Database do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  alias TrentoWeb.OpenApi.Schema.{ResourceHealth, Tags}
+
+  defmodule DatabaseInstance do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "DatabaseInstance",
+      description: "A discovered HANA Database Instance on the target infrastructure",
+      type: :object,
+      properties: %{
+        sap_system_id: %Schema{type: :integer, description: "SAP System ID", format: :uuid},
```

Comment:
Is this `type: :integer` correct? Shouldn't be `:string` as at the end it is a uuid

Reply:
changed